apps/www/_blog/2024-07-11-hardening-supabase.mdx

```diff
@@ -0,0 +1,169 @@
+
+- _OpenAPI management_
+
+  We will be giving developers the ability to control the visibility of the OpenAPI spec based on the Postgres roles and grants. This provides some level of security through obscurity.
+
+- _Development on "Hard mode"_
+
+  Not all projects are alike. While some require extremely strict levels of access, others can be a bit more relaxed. This is explained in our [Maturity Models](https://supabase.com/docs/guides/platform/maturity-model) framework. We are brainstorming ways to codify this, so that developers can “turn up” the enforced security levels on their projects, restricting what developers can and cannot do.
+
+- _CI/CD warnings_
+
+  We will be adding the [Security Advisories](https://github.com/supabase/splinter) into our CLI for first-class support within GitHub Actions and CI/CD pipelines. Combined with [Branching](https://supabase.com/docs/guides/platform/branching), this prevents security misconfigurations before they ever reach production.
+
+- _Revamped API keys_
+
+  Soon we will be replacing the `anon` and `service_role` API keys with a simpler `public` and `admin` key. This new setup will be conceptually similar, but will allow developers to create more fine-grained authorization rules and allow them to roll individual keys in case of a leak.
+
+## Supabase Responsible Disclosure Program
+
+Eva approached us during our last Launch Week. While she was helping us with the Security Advisor she discovered a misconfiguration in one of our own applications using Supabase with [Lunchcat](https://lunchcat.dev/). We were able to solve this quickly by toggling off read access to the `anon` role and scanning the [access logs](https://supabase.com/docs/guides/platform/logs) to ensure no malicious actors had accessed the data.
+
+At the time, we didn’t have the Security Advisor to help the developer discover the misconfiguration before publishing. The Supabase Security suite is being developed to prevent this ever occurring again - for us and for our customers.
+
+But no platform is infallible: when tooling doesn’t work, we rely on our community to help discover and responsibly disclose any vulnerabilities. We are launching a private Vulnerability Disclosure Program today with HackerOne. We commit to transitioning to a public disclosure program in a month, once we iron out the initial kinks. If you cannot wait till then, use our [secret link](https://hackerone.com/ca63b563-9661-4ac3-8d23-7581582ef451/embedded_submissions/new) submit a report.
+
+If you find any misconfigured projects, we will work with those customers to ensure that they know about the issue and can solve it. If you are a security professional, we welcome your help to secure the Supabase community.
```

Comment:
The phrasing is awkward here. Are we trying to say "If *we* find any misconfigured projects..." or are we saying, "If you find a misconfigured project, please let us know and we will work..."? Suggest changing it to one or the other to make it clear what we are trying to say because right now we have a grammatical issue with the way it's written.